lasagne/layers/shape.py

```diff
@@ -67,6 +69,12 @@ class ReshapeLayer(Layer):
     >>> l1 = ReshapeLayer(l_in, ([0], [1], 5, -1))
     >>> l1.output_shape
     (None, 100, 5, 4)
+    >>> x = T.tensor3()  # reshape depending on input size
+    >>> batch_size, seq_len, _ = x.shape
+    >>> l_in = InputLayer((None, None, 10))
+    >>> l1 = ReshapeLayer(l_in, (batch_size*seq_len, [2]))
+    >>> l1.output_shape
+    (None, 10)
```

Comment:
That's not a good example, because you could just do `ReshapeLayer(l_in, (-1, [2]))` for that. Better remove the example than keeping people wondering.